knowledge/musical_instruments/les_paul_studio/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: musical_instruments
+seed_examples:
+- answer: The Studio Custom was produced in 1983–1985 |
+  question: What year was the Studio Custom produced
+- answer: In 2009, the Vintage Mahogany was renamed the Les Paul Studio Faded |
+  question: What is Vintage Mahogany/Faded 
+- answer: The Studio Pro Plus was an upgraded studio with a AA figured maple top and gold hardware. |
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!